### DISCUSSION TITLE
Email change for institution

### DISCUSSION THREAD

**Discussion by pr72ldp** [community/community#48490] [GitHub Education] [answered]
Hello, being a student at ETML in Lausanne, Github Education doesn't seem to allow me to register. It still expects the old emails addresses "@etml.educanet2.ch" which were discontinued after the switch to "@eduvaud.ch". What could be done to update them? Thanks!

**Comment by samus-aran** [comment, marked as answer]
Hi @pr72ldp 👋 

We've reviewed this particular school and have updated those domains for you. Please retry again and let us know how you get on 👍

**Reply by pr72ldp** [reply]
Thanks a lot, I checked and it now works, have a nice day!

**Comment by luarredondo** [comment]
hello ! I will change my email but i don't have a "resend verrification email" to my new email.

**Comment by oktay-kagan** [comment]
Hello! I am a student at Istanbul Aydin University. I unlinked my school email from my old GitHub account because I can't use that account. So I created a new account and added school mail. But it says it is already using on my old account. Could you please help me to solve this problem?
